registrations/tasks.py

```diff
@@ -273,8 +274,10 @@ def create_subscriptionrequests(self, registration):
         weeks = 1  # default week number
 
         # . calculate weeks along
-        if (registration.reg_type == "momconnect_prebirth" and
-           registration.source.authority not in ["hw_partial", "patient"]):
+        if (registration.reg_type in [
+                "momconnect_prebirth", "whatsapp_prebirth"] and
+                registration.source.authority not in [
+                    "hw_partial", "patient"]):
```

Comment:
I'm not sure that this is the correct place for it. Since we've added the whatsapp choice to the public line, the authority will always be `patient`, so this if statement will never be true for whatsapp registrations.